src/content/docs/tutorial-create-alerts/create-chart-alerts.mdx

```diff
@@ -0,0 +1,121 @@
+
+3. Modify the NRQL query to match what you want you to monitor, if necessary. For now, we recommend leaving the query as is, but once you have a better handle on alerts and NRQL queries, you can change the query to monitor nearly any data threshold
+
+4. Select **Next**
+    </Step>
+    <Step>
+## Adjust the signal behavior
+
+<img
+    title="Tuning your signal"
+    alt="A screenshot displaying how to set options for a signal in an alert condition"
+    src={alertsTune}
+/>
+
+1. Set the data aggregation options. You will need to:
+    * Use the **Window duration** setting to determine how frequently we aggregate your data, such as groupings of every 5 minutes or every hour
+    * Choose whether or not you want to use [sliding window aggregation](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#sliding-window-aggregation). This feature creates smoother charts when dealing with erratic data by creating overlapping  aggregation windows. This feature is disabled by default, and we recommend keeping it disabled for your first alert
+    * Choose the streaming method you want to use. The two options most often used are **Event flow** and **Event timer**. **Event flow** is best suited for frequent and consistently reporting data, while **Event timer** is best suited for data that reports inconsisntely, infrequently, or ingests in batches like (example). Choose the option that best fits the behavior of your monitored data. If you want to learn more about the individual settings to see which method should be used in each scenario, see [our docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/streaming-alerts-key-terms-concepts/#aggregation-methods)
+
+2. Set the timing option for your aggregation method, which determines how long we wait for events in each window before we and evaluate it. Longer delays result in more accurate alerts, but will also delay the opening of incidents. This option is called **Delay** in the **Event flow** and **Cadence** options and **Timer** in **Event timer**. For your first alert, we recommend keeping the default time setting.
+
+3. Select the gap filling strategy for this alert. Gap-filling is a reactive feature that fills blank periods in data reporting with user-defined synthetic value. If you believe the data reporting will be sporadic, you can avoid false alerts by filling in empty aggregation windows with this data. For your first alert, select **None** under **Gap-filling strategy**.
+
+4. Choose whether or not to use **Evaluation delay**. This option enables a custom period that New Relic waits before starting to evaluate a newly ingested signal against the threshold you set. It's useful for preventing false positive incidents when a data stream begins to report. For your first alert, you can leave it to the disabled toggle unless you know that this option applies to your data.
```

Comment:
```This option enables a custom period that New Relic waits before starting to evaluate a newly ingested signal against the threshold you set```

This is a pretty complex sentence.

Reply:
I've done my best to simplify, but this is a complicated feature with a lot going on. Let's discuss further.